src/ut/handlers_test.cpp

```diff
@@ -4140,6 +4237,43 @@ TEST_F(HandlerStatsTest, DigestCacheFailure)
   delete _caught_diam_tsx; _caught_diam_tsx = NULL;
 }
 
+TEST_F(HandlerStatsTest, DigestCacheConnectionFailure)
+{
+  // Test that UNsuccessful cache requests result in the latency stats being
```

Comment:
Shouldn't this test just ignore latency and check for a 503, so it's only testing one thing?